model/vm.rb

```diff
@@ -210,7 +210,7 @@ def params_json(swap_size_bytes)
       "public_ipv4" => ip4.to_s || "",
       "local_ipv4" => local_vetho_ip.to_s.shellescape || "",
       "unix_user" => unix_user,
-      "ssh_public_key" => public_key,
+      "ssh_public_keys" => [public_key],
```

Comment:
Should we move the rhizome change to a prior commit and deploy it first?

Currently, when we deploy it, clover will send `ssh_public_keys` to the rhizome but it expects `ssh_public_key` until we run `install_rhizome` for all hosts.

Reply:
Makes sense. I put the rhizome changes to the first commit.